quipucords/scanner/openshift/metrics.py

```diff
@@ -0,0 +1,31 @@
+"""OpenShift Prometheus metrics definitions and utilities."""
+
+OCP_PROMETHEUS_METRICS = {
+    "node_metrics": {
+        "attributes": [
+            "instance",
+            "label_node_hyperthread_enabled",
+            "label_node_role_kubernetes_io_master",
```

Comment:
do we actually need `label_node_role_kubernetes_io_master`?

Reply:
Good point! I have removed this in a subsequent push.

Reply:
I am assuming that whatever sets these labels (I believe they are populated automatically, not by user input) is driven by each node's configuration, and hyperthreading can be configured independently on each node.